modules/dheap/include/dheap.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 Gogov Vladislav
+
+#include <iostream>
+#include <vector>
+
+#ifndef MODULES_DHEAP_INCLUDE_DHEAP_H_
+#define MODULES_DHEAP_INCLUDE_DHEAP_H_
+
+class Dheap {
+    int base;
+    std::vector<int> weight;
```

Comment:
Логичнее назвать поле `weights`.
```suggestion
    std::vector<int> weights;
```